### PR TITLE
update docker scripts

### PR DESCRIPTION
**Pull request by jesterhazy**
*Issue #, if available:* #312 

*Description of changes:*

- update `install.sh` so image build runs correctly
- update gpu image so it has version of cuda required by mxnet-cu90

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/Dockerfile.gpu [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:8.0-cudnn5-devel
+FROM nvidia/cuda:9.0-cudnn7-devel
```
Comment (lupesko) [inline docker/Dockerfile.gpu]: Have we tested the GPU container?

File: docker/install.sh [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
-apt-get install protobuf-compiler libprotoc-dev
 
-pip install --upgrade pip
-pip install gunicorn
-pip install gevent
-pip install mxnet-model-server
+apt-get install -y build-essential libatlas-base-dev libopencv-dev graphviz python-dev \
+    openjdk-8-jdk nginx protobuf-compiler libprotoc-dev python-certbot-nginx curl
+
+cd /tmp && curl -O https://bootstrap.pypa.io/get-pip.py && python get-pip.py
```
Comment (lupesko) [inline docker/install.sh]: hmm. seems a bit unclean... why not something along the lines of:
```
$ apt-get install python-pip 
$ pip install --upgrade pip 
```

Comment (jesterhazy) [reply docker/install.sh]: apt-get repo may not have latest version of pip (usually doesn't). this is the best way to install pip.

File: docker/Dockerfile.gpu (hunk 1 above)
Comment (jesterhazy) [reply docker/Dockerfile.gpu]: yes

File: docker/install.sh (hunk 2 above)
Comment (lupesko) [reply docker/install.sh]: Got it. Cool.

**Pull request merged**